src/rTerminal.ts

```diff
@@ -237,7 +237,7 @@ export async function runChunksInTerm(chunks: vscode.Range[]): Promise<void> {
     }
 }
 
-export async function runTextInTerm(text: string): Promise<void> {
+export async function runTextInTerm(text: string, execute = true): Promise<void> {
```

Comment:
Suggest adding parameter type to function signature.

```suggestion
export async function runTextInTerm(text: string, execute: boolean = true): Promise<void> {
```